aesthetic_ants/penguin/enemy.py

```diff
@@ -2,24 +2,50 @@
 
 from .snowball import Snowball
 from .constants import CollisionType
+from .object import Object
 from .object import PhysicalObject
-from .resources import ENEMY_IMAGE
+from .resources import ENEMY_BIG_IMAGE, ENEMY_FAST_IMAGE
 
 
 class Enemy(PhysicalObject):
     collision_type = CollisionType.ENEMY
     score = 5
 
-    def __init__(self):
-        super().__init__(ENEMY_IMAGE)
-
-        self.velocity_x = random.randint(0, 100)
-        self.velocity_y = random.randint(0, 100)
+    def __init__(self, x, y, speed, hearts, enemy_image):
+        super().__init__(enemy_image, x, y)
+        self.velocity_x = speed
+        self.velocity_y = speed
+        self.hearts = hearts
+        self.hit_count = 0
 
     def update(self, dt):
-        self.x += self.velocity_x * dt
-        self.y += self.velocity_y * dt
+        None
+        #self.x += self.velocity_x * dt
+        #self.y += self.velocity_y * dt
 
     def on_collision_snowball(self, snowball: Snowball):
-        self.space.remove(self)
+        self.hit_count += 1
+        if self.hit_count == self.hearts:
+            self.space.remove(self)
         self.space.remove(snowball)
+
+
+class BigEnemy(Enemy):
+
+    def __init__(self):
```

Comment:
Keep the `__init__` consistent -- it should take `(self, x, y)` for its arguments.